deployment/eks/flyte_aws_scheduler_helm_generated.yaml

```diff
@@ -492,10 +491,7 @@ data:
         - k8s-array
   k8s.yaml: | 
     plugins:
-      agent-service:
-        defaultGrpcEndpoint: flyteagent.flyte.svc.cluster.local:8000
-        supportedTaskTypes:
-        - bigquery_query_job_task
+      agent-service: {}
```

Comment:
Should we remove this variable completely from the rendered template if `agent-service` is not set?

Reply:
good call, I'll make it part of the comment if people want a model of how to deploy this before the agents helm chart lands.